public/js/cart.js

```diff
@@ -128,6 +124,5 @@ function convertToSlug(Text)
     return Text
         .toLowerCase()
         .replace(/ /g,'-')
-        .replace(/[^\w-]+/g,'')
-        ;
+        .replace(/[^\w-]+/g,'');
 }
```

Comment:
Please add an empty line at the end of the file, you should config your editor to auto adds a new line when save.
Ref: https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files